conans/client/graph/graph.py

```diff
@@ -219,6 +219,7 @@ def serialize(self):
         result["binary"] = self.binary
         # TODO: This doesn't match the model, check it
         result["invalid_build"] = self.cant_build
+        result["info_invalid"] = self.conanfile.info.invalid if self.conanfile.info is not None and self.conanfile.info.invalid is not None else False
```

Comment:
Why ``None`` isn't good here, instead of ``False``? If there is no information about it, I'd expect ``None`` as lack of existence, better than False?

Reply:
Because `invalid_build` seems to work that way, so I made it behave like it, but I'd also prefer having None here if it weren't for that

Reply:
``invalid_build`` gets the value from ``cant_build``, and in this case it is find to be ``False``, you don't lack the information, you know if it is either valid or invalid (boolean), as opposed to the ``info_invalid`` which is string (or None if not defined)

Reply:
You're right! I'll change it